include/rocksdb/listener.h

```diff
@@ -206,6 +230,22 @@ struct FileOperationInfo {
   }
 };
 
+struct BlobFileInfo {
+  std::string blob_file_path;
+  uint64_t blob_file_number;
+  uint64_t total_blob_count;
+  uint64_t total_blob_bytes;
+  CompressionType compression_type;
```

Comment:
We could simply store `blob_compression_type` once as part of `FlushJobInfo` / `CompactionJobInfo`; we don't have to expose it on a per-file basis because it's guaranteed to be the same for all bllob files.